src/frontend/dpia-webapp/src/components/TheWelcome.vue

```diff
@@ -1,10 +1,10 @@
-<script setup>
-import WelcomeItem from './WelcomeItem.vue'
```

Comment:
These files are default files when we create starter vue project. Eventually we would need to delete them. Such an architecture by default calls for single point of failure.

Reply:
It's part of the hello world you can see served in openshift. We can delete the components when we actually start development.